ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -369,4 +367,76 @@ private Schema buildAggregateSchema(
     return schemaBuilder.build();
   }
 
+
+  class InternalSchema {
```

Comment:
At some point I think we should pull this out and pass it around to the KudafAggregator and KsqlAggregateFunctions. That way we can encapsulate away some of these mappings. I'm fine with doing that in another PR though.